### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by shamansir**

### CODE REVIEW

**Comment by shamansir** [comment]
Please do not merge yet, I'm testing Travis CI Integration.

File: spec/check.spec.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+describe('test', function() {
+
+    it('works', function() {
+        expect(true).toBeTruthy();
```
Comment (mcm69) [inline spec/check.spec.js]: :+1:

**Comment by shamansir** [comment]
Oh. It worked. Nice!

File: src/anm/animation/animation.js [hunk 2]
```diff
@@ -389,7 +389,8 @@ Animation.prototype.handle__x = function(type, evt) {
                 }
             });
         });
-        if ((type === 'mousemove') && !foundTarget && anim.__lastOverElm) {
+        if ((type === 'mousemove') && !foundTarget &&
+            anim.__lastOverElm && anim.__lastOverElm.subscribedTo('mouseout')) {
             var stillInside = false;
```
Comment (shamansir) [inline src/anm/animation/animation.js]: Mmm, anyway it's here. Okay

**Pull request merged**